src/languageSetup.ts

```diff
@@ -18,14 +18,14 @@ import { MainClassRequest, OverrideMemberRequest } from "./lspExtensions";
 export async function activateLanguageServer({ context, status, config, javaInstallation, javaOpts }: ServerSetupParams): Promise<KotlinApi> {
     LOG.info('Activating Kotlin Language Server...');
     status.update("Activating Kotlin Language Server...");
-    
+
```

Comment:
I think it's fine here, but in general, please try to avoid including unrelated formatting changes in commits.

Reply:
Yeah, that was auto-remove of trailing whitespaces in ide. Sorry about that.

Should i revert this or let it pass?